### DISCUSSION TITLE
[Question] Disable verbose logging

### DISCUSSION THREAD

**Discussion by StoneT2000** [isaac-sim/IsaacLab#1097] [Q&A] [answered]
Is there a way to disable all the logging from Isaac? I could not find some documentation on the carb package and how to change the settings.

Thanks!

**Comment by Dhoeller19** [comment]
Hi @StoneT2000, unfortunately, this is not yet possible.

**Comment by Mayankm96** [comment, marked as answer]
I took a step towards this in #1217. The remaining prints are coming from Isaac Sim's `SimulationApp` which can hopefully also be removed to have a nice clean console output.

**Reply by reeceomahoney** [reply]
> I took a step towards this in #1217. The remaining prints are coming from Isaac Sim's `SimulationApp` which can hopefully also be removed to have a nice clean console output.

Is removing the SimulationApp prints already possible or planned for a future release?
